vitest.config.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: ['./src/test/setup.ts'],
+    coverage: {
+      exclude: ['**/*.stories.tsx', '**/*.stories.ts'],
+    },
```

Comment:
Pls also exclude storybook static folder, node_modules, etc.
Or it is better to provide the dir to collect coverage from.

Pls also apply the test coverage threshold, it should be 80%. But to make the build passed, pls, temporary collect coverage from the Button component only.